src/api/app/assets/javascripts/webui/notification.js

```diff
@@ -0,0 +1,32 @@
+function setSelectAllCheckbox() {
+  $('#select-all-notifications').change(function() {
+    var checkboxes = $(this).closest('form').find('input[type=checkbox]');
+    checkboxes.prop('checked', $(this).is(':checked'));
+  })
+}
+
+function setCheckboxCounterAndSubmitButton() {
+  var amountBoxesChecked = 0;
+  $('input[type="checkbox"]').each(function() {
+    if($(this).is(':checked')){
+      amountBoxesChecked += 1;
+    }
+  });
+
+  if(amountBoxesChecked <= 0) {
+    $('#done-button').addClass('d-none');
+    $('#select-all-label').text('Select all');
+  } else {
+    $('#done-button').removeClass('d-none');
+    $('#select-all-label').text('selected ' + amountBoxesChecked);
```

Comment:
It reads much better like this:

```suggestion
    $('#select-all-label').text(amountBoxesChecked + ' selected');
```